pyquil/quil.py

```diff
@@ -951,7 +984,9 @@ def implicitly_declare_ro(instructions: List[AbstractInstruction]):
     return new_instr
 
 
-def merge_with_pauli_noise(prog_list: Iterable, probabilities: List, qubits: List) -> Program:
+def merge_with_pauli_noise(
+    prog_list: Iterable[Program], probabilities: List[float], qubits: Sequence[Any]
```

Comment:
```suggestion
    prog_list: Iterable[Program], probabilities: Sequence[float], qubits: Sequence[Any]
```

`pauli_kraus_map` is declare to take `Sequence[float]` so I think this should be ok?

Reply:
Fixed in 5th commit.